shpkpr/marathon/schema/app.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
nooooo, it's the official schema from the docs :) https://mesosphere.github.io/marathon/docs/generated/api.html#v2_apps__app_id__put

Need to add some attribution somewhere, not sure where.